app/manager-frontend/public/locales/dev/core.json

```diff
@@ -119,5 +119,19 @@
       "loggedIn": "You have been logged in.",
       "invalidLink": "The password reset failed. Please request a new reset link, or contact support."
     }
-  }
+  },
+  "homepageActions": [
```

Comment:
this file isn't a good place for this.

the files in `locales` are only for text translations, and they're in the `public/` folder as they're intended to be served statically by the site, so they're not appropriate as source code for importing into other files.

If these actions are only used on the homepage, just keep them in there. if they're shared then they probably want to go in a `config/` or `data/` folder? in their own module

Reply:
I will move them in the home page (UserHome) as they are not shared.